src/exercise/04.html

```diff
@@ -8,17 +8,120 @@
   <script src="https://unpkg.com/@babel/standalone@7.12.4/babel.js"></script>
   <script type="text/babel">
     // 🐨 Make a function called `message` which returns the JSX we want to share
+    const message = ({children}) => {
+      return <div className="message">{children}</div>;
+    }
 
     // 🐨 use that function in place of the divs below with:
     // 💰 {message({children: 'Hello World'})} {message({children: 'Goodbye World'})}
     const element = (
       <div className="container">
-        <div className="message">Hello World</div>
-        <div className="message">Goodbye World</div>
+        {message({children: 'Hello World'})} {message({children: 'Goodbye World'})}
       </div>
     )
 
     // 💯 This is only the first step to making actual React components. The rest is in the extra credit!
     ReactDOM.render(element, document.getElementById('root'))
   </script>
+
+<!--EXTRA CREDIT 1-->
+<div id="extra-1"></div>
```

Comment:
I like that you included all the extra credit separately, I just iterated on the same one.

Reply:
Glad you like it! I was afraid someone would be like ARGH WHY. But I like to keep all of it for reference.